resources/deu.ini

```diff
@@ -271,7 +278,7 @@ clone.name=clone
 clone.desc=Klont ein Grundstück an eine andere Position
 clone.usage=/p clone <id: string> [welt: string]
 clone.wrongid=Die Grundstück ID sollte in diesem Format sein: X;Z
-clone.nomoney=Du besitzt nicht genug Geld um dieses Plot zu klonen
+clone.nomoney=Du besitzt nicht genug Geld um dieses Grundstück zu klonen
```

Comment:
This change is outside the scope of this PR

Reply:
Oops, you're right.